tests/compile/zinc-inc/test

```diff
@@ -4,6 +4,6 @@
 echo "class A$RANDOM" > Example.gen.scala
 
 rm -fr "$(bazel info execution_root)/.bazel-zinc"
-bazel build --noworker_sandboxing --worker_extra_flag=ScalaCompile=--persistence_dir=.bazel-zinc :lib
+bazel build --experimental_output_paths=off --noworker_sandboxing --worker_extra_flag=ScalaCompile=--persistence_dir=.bazel-zinc :lib
```

Comment:
Why'd we turn this setting off?

Reply:
I do not remember and will poke at this. I think this test fails with path mapping on, but I could be wrong about that.

In general the incremental Zinc compilation tests just disable everything like path mapping, sandboxing, etc. in order to function.

That said, now that I know everything else is working I'll go try the test with path mapping on and see what happens.

Reply:
Sounds good!

Reply:
Confirmed that path mapping needs to be disabled for the incremental compilation stuff to work at all as it reaches outside Bazel's sandbox and thus things like path mapping trip it up.

I have some code in phase_zinc_compile that should disable path mapping if using a toolchain that has incremental compilation enabled. Problem is, we're just using the default toolchain in this test and using command line options to disable sandboxing and path mapping.

I don't think I care enough about the incremental compilation stuff to improve this as we're likely ripping that feature out soon.